web/src/test/java/org/mskcc/cmo/metadb/web/RestIntegrationTest.java

```diff
@@ -0,0 +1,98 @@
+package org.mskcc.cmo.metadb.web;
+
+import static org.junit.Assert.assertEquals;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.json.JsonParser;
+import org.springframework.boot.json.JsonParserFactory;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.web.client.RestTemplate;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class RestIntegrationTest {
+    private final static String BASE_URL = "http://localhost:8080/request/";
+    
+    private RestTemplate restTemplate = new RestTemplate();
+
+    private Map<String, Object> fetchMetaDbRequestGET(String requestId)
+    {
+        String response = this.restTemplate.getForObject(BASE_URL + requestId, String.class);
+        JsonParser springParser = JsonParserFactory.getJsonParser();
+        Map<String, Object> map = (Map<String, Object>)(Map) springParser.parseMap(response);
+        return map;
+    }
+
+    private List<Map<String, Object>> fetchMetaDbRequestListPOST(String[] requestIds)
+    {
+        String responses = this.restTemplate.postForObject(BASE_URL, requestIds, String.class);
+        JsonParser springParser = JsonParserFactory.getJsonParser();
+        List<Map<String, Object>> lists = (List<Map<String, Object>>)(List<?>) springParser.parseList(responses);
+        return lists;
+    }
+    
+    @Test
+    public void testMetaDbRequestFetch() {
```

Comment:
Is the test actually establishing a connection to the neo4j on dashidev?

If so then we should add an embedded neo4j graphdb (can you see if this is possible?) I'm sure there are others who want to test the same sort of functionality.. 

Anyway, if we can't create and populate a small neo4j graphdb in memory during testing then an alternative would be to use Mockito to mock responses [see genome nexus annotator example here](https://github.com/genome-nexus/genome-nexus-annotation-pipeline/blob/master/annotator/src/test/java/org/cbioportal/annotator/MockGenomeNexusImpl.java)

Reply:
mock data can be loaded from the test data files that we checked into the repo: https://github.com/mskcc/cmo-metadb/tree/master/test/data

Reply:
one basic test case: mock data loaded from one of the the json files linked above and check that it's deserialized correctly (fields match the expected values)